src/averageAges.js

```diff
@@ -1,61 +1,50 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function calculateAverageAge(persons) {
+  const ages = persons.reduce(
+    (sum, person) => sum + (person.died - person.born),
+    0
+  );
```

Comment:
You may try destructuring here
```suggestion
  const ages = persons.reduce((sum, { died, born }) => (
    sum + died - born
   ) , 0);
```